recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - osx-dist.patch       # [osx]
+    - win-find_exe.patch   # [win]
+
+build:
+  no_link: 
+    - bin/python
+  number: 0
+
+requirements:
+  build:
+    - bzip2     # [unix]
+    - zlib      # [unix]
+    - sqlite    # [unix]
+    - readline  # [unix]
+    - tk        # [unix]
+    - openssl   # [unix]
+    - xz        # [unix]
+    - ncurses   # [linux]
```

Comment:
We can build svn without python dependency if it helps. I put it in so that people could interact with svn from python, but I don't think that's going to see much use.